controllers/ext_client.go

```diff
@@ -648,3 +643,35 @@ func doesUserOwnClient(username, clientID, network string) (bool, bool) {
 
 	return false, logic.StringSliceContains(netUser.Clients, clientID)
 }
+
+// validateExtClient	Validates the extclient object
+func validateExtClient(extclient *models.ExtClient, customExtClient *models.CustomExtClient) error {
+	//validate clientid
+	if customExtClient.ClientID != "" && !validName(customExtClient.ClientID) {
```

Comment:
`if customExtClient.ClientID == "" || !validName(customExtClient.ClientID)`?
if the id is empty or is not valid

Reply:
only validate if a client name was provided.  it is optional